### PR TITLE
IALERT-3413 Remove signed assertions

### PR DESCRIPTION
**Pull request by mau-synopsys**
Remove the option to sign assertions to be in line with library implementation.

A new migration script need to be added to drop the column from the database.

### CODE REVIEW

File: alert-database/src/main/resources/liquibase/6.13.2/changelog.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (psantos1113) [inline alert-database/src/main/resources/liquibase/6.13.2/changelog.xml:1]: put this changelog file in a 7.0.0 directory.

File: alert-database/src/main/resources/liquibase/changelog-postgres-master.xml [hunk 2]
```diff
@@ -26,4 +26,5 @@
     <include file="6.12.0/changelog.xml" relativeToChangelogFile="true"/>
     <include file="6.13.0/changelog.xml" relativeToChangelogFile="true"/>
     <include file="6.13.1/changelog.xml" relativeToChangelogFile="true"/>
+    <include file="6.13.2/changelog.xml" relativeToChangelogFile="true"/>
```
Comment (psantos1113) [inline alert-database/src/main/resources/liquibase/changelog-postgres-master.xml:29]: change the file path to 7.0.0/changelog.xml

**Review by DanaMaxfield — APPROVED** [review]
Approved assuming edits from Paulo

**Comment by mau-synopsys** [comment]
Moved liquibase changes to 7.0.0 directory

**Review by psantos1113 — APPROVED** [review]

**Pull request merged**